packages/concerto-tools/lib/codegen/fromcto/csharp/csharpvisitor.js

```diff
@@ -283,8 +284,17 @@ class CSharpVisitor {
 
         let fieldType = externalFieldType ? externalFieldType : this.getFieldType(field);
 
+        if (fieldType === 'String') {
```

Comment:
I worry that we might be using an `externalFieldType` here...

Reply:
yes, we might. But external type can be UUID scalar type. We don't want to add regex decorator for UUID.